app/javascript/stylesheets/application.scss

```diff
@@ -1,6 +1,12 @@
 @import "~bootstrap/scss/bootstrap";
 @import "./_sidebar";
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
 
-#page-content-wrapper {
-  width: 100%;
+.column-background {
```

Comment:
Consider renaming this to `.authentication-landing-column` or similar, to make it more descriptive.

Reply:
That's good idea, thanks.